presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2347,6 +2350,40 @@ public void testInvalidPartitionValue()
                 "\\QHive partition keys can only contain printable ASCII characters (0x20 - 0x7E). Invalid value: E2 98 83\\E");
     }
 
+    @Test
+    public void testCurrentUserInView()
+    {
+        if (!getSession().getCatalog().isPresent() || !getSession().getSchema().isPresent()) {
+            throw new SkipException("Session catalog and schema must be set");
```

Comment:
Do we actually want to skip here, or should this be an error? When would these not be set?